packages/kbn-es-archiver/src/lib/indices/create_index_stream.test.ts

```diff
@@ -212,18 +216,63 @@ describe('esArchiver: createCreateIndexStream()', () => {
       // So, we first delete only the Kibana task manager indices, then we wind up deleting all Kibana indices.
       await doTest('.kibana_task_manager_7.16.0_001', '.kibana_7.16.0_001');
 
-      expect(mockDeleteKibanaIndices).toHaveBeenCalledTimes(2);
-      expect(mockDeleteKibanaIndices).toHaveBeenNthCalledWith(
+      expect(mockDeleteSavedObjectIndices).toHaveBeenCalledTimes(2);
+      expect(mockDeleteSavedObjectIndices).toHaveBeenNthCalledWith(
         1,
         expect.objectContaining({ onlyTaskManager: true })
       );
-      expect(mockDeleteKibanaIndices).toHaveBeenNthCalledWith(
+      expect(mockDeleteSavedObjectIndices).toHaveBeenNthCalledWith(
         2,
-        expect.not.objectContaining({ onlyTaskManager: true })
+        expect.not.objectContaining({ index: expect.any(String) })
       );
     });
   });
 
+  describe('saved object cleanup', () => {
+    describe('when saved object documents are found', () => {
+      it('cleans the corresponding saved object indices', async () => {
+        const client = createStubClient();
+        const stats = createStubStats();
+        await createPromiseFromStreams([
+          createListStream([
+            createStubDocRecord('.kibana_task_manager', 1),
+            createStubDocRecord('.kibana_alerting_cases', 2),
```

Comment:
For myself: we do not have this SO index in 7.17.x? I understand it is just a test and won't affect actual code.

Reply:
Good catch, we shouldn't be introducing these 8.8 spoilers in 7.x.
I'll update the test.